src/app/app.module.ts

```diff
@@ -1,14 +1,17 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
-
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { AdminLayoutComponent } from './admin-layout/admin-layout.component';
 import { SidebarComponent } from './sidebar/sidebar.component';
 import { NavbarComponent } from './navbar/navbar.component';
 import { RabbitmqService } from './shared/services/rabbitmq.service';
 import { HttpClientModule } from '@angular/common/http';
+import { SocketIoModule, SocketIoConfig } from 'ngx-socket-io';
+import { FormsModule } from '@angular/forms';
+import { ApiService } from './shared/services/api.service';
 
+const config: SocketIoConfig = { url: 'http://localhost:3000', options: {} };
```

Comment:
Will this hard coded values cause issues in a docker container?

Reply:
Let me look into that one.  I'll build the docker container locally and see how it works.  Docker env variables are not as easily done within Angular.  You either need to mount a volume or replace a JSON file.

Reply:
This hardcoded value does not cause issues in a docker container.  since the ui container is exposing port 3000 to the host, the path to localhost:3000 is correct in this case.